lib/ast.js

```diff
@@ -589,61 +589,64 @@ var AST_Seq = DEFNODE("Seq", "car cdr", {
         cdr: "[AST_Node] second element in sequence"
     },
     $cons: function(x, y) {
+        if (!x) {
+            throw new Error('AST_Seq.car missing');
+        }
```

Comment:
I put it there to aid testing, i.e. if your throw code snipplets as you usually do at this PR, it will error out quickly and clearly.

So let's remove that at the end of this PR. But good point about documentation - I always forgot to add those... 😓